backend/open_webui/utils/telemetry/metrics.py

```
@@ -0,0 +1,110 @@
+* http.server.duration (histogram, milliseconds)
+
+Attributes used: http.method, http.route, http.status_code
+
+If you wish to add more attributes (e.g. user-agent) you can, but beware of
+high-cardinality label sets.
+"""
+
+from __future__ import annotations
+
+import time
+from typing import Dict, List, Sequence, Any
```

Ruff (0.11.9)

Import from `collections.abc` instead: `Sequence`

Import from `collections.abc`

(UP035)